PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PowerPointLabs.TimerLab
+{
+    internal static class NotesToTimer
```

Comment:
May I know why this class is called NotesToTimer? Or is there any better place to put IsTimerEnabled?

Reply:
@LiXiaoooowei Would you be updating this PR?

Reply:
@initialshl I am trying to follow the names set by other labs for similar functionalities, i.e. `NotesToAudio` and `NotesToCaption`.

Reply:
Hmm I think that the `NotesToAudio` and `NotesToCaption` refers to the slide's speaker's notes (text-to-speech). You may want to change it to just `TimerLab.cs`

Reply:
@initialshl Code edited according to suggestion.